src/libraries/System.Diagnostics.StackTrace/src/ILLinkTrim.xml

```diff
@@ -1,7 +0,0 @@
-<linker>
-  <assembly fullname="System.Diagnostics.StackTrace">
-    <!-- used by System.Private.CoreLib StackTrace code to load portable pdbs for the runtime diagnostic stack trace to get source/line info -->
-    <type fullname="System.Diagnostics.StackTraceSymbols" required="true" />
```

Comment:
Is the per-assembly linking used by CoreFX build going to keep the type around without this?

Reply:
I think it will as libraries are built with coreclr by default for packaging but this raises good point about handling runtime specific dependencies during libraries illinker optimization pass.

Reply:
The failures in `System.Tests.ExceptionTests.VerifyCallStack` tests that are popping up in the CI suggest that the type is not kept around.

Reply:
good catch - I reverted the deletion of `ILLinkTrim.xml`.